tests/integration/telemetry/stackdriver/vm/main_test.go

```diff
@@ -59,7 +127,14 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
 		RequireSingleCluster().
-		Setup(istio.Setup(&i, func(_ resource.Context, cfg *istio.Config) {
+		Setup(istio.Setup(&istioInst, func(_ resource.Context, cfg *istio.Config) {
+			cfg.Values["meshConfig.enableTracing"] = "true"
+			cfg.Values["meshConfig.defaultConfig.tracing.sampling"] = "100.0"
+			cfg.Values["meshConfig.defaultConfig.tracing.custom_tags.canonical_service_name.environment.name"] = "CANONICAL_SERVICE"
+			cfg.Values["meshConfig.defaultConfig.tracing.custom_tags.canonical_service_name.environment.defaultValue"] = "unknown"
+			cfg.Values["meshConfig.defaultConfig.tracing.custom_tags.canonical_service_revision.environment.name"] = "CANONICAL_REVISION"
+			cfg.Values["meshConfig.defaultConfig.tracing.custom_tags.canonical_service_revision.environment.defaultValue"] = "latest"
```

Comment:
should this be set to `earliest` too as in envTagsProxyConfig?

Reply:
great catch. updated.